metadata-ingestion/src/datahub/ingestion/source/unity/proxy_types.py

```diff
@@ -170,10 +172,10 @@ class Query:
     end_time: datetime
     # User who ran the query
     user_id: int
-    user_name: str  # Email or username
+    user_name: Optional[str]  # Email or username
```

Comment:
Turns out this can be empty if you delete the user that made the query... was causing errors, so now I just don't report the user in the operation / usage statistics aspect if we dont' have it